ftd/t/js/53-link-color.html

```diff
@@ -16,9 +16,10 @@
     </style>
 </head>
 <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
-<body data-id="1"><div data-id="2" class="ft_column __w-1 __h-2"><a data-id="3" href="https://google.com">Click me</a><div data-id="4">Hello world [Test](https:&#47;&#47;google.com)
-
-This is awesome [Test](https:&#47;&#47;google.com)</div></div></body><style id="styles">
+<body data-id="1"><div data-id="2" class="ft_column __w-1 __h-2"><a data-id="3" href="https://google.com">Click me
+</a><div data-id="4">&lt;p&gt;Hello world &lt;a href=&quot;https:&#47;&#47;google.com&quot;&gt;Test&lt;&#47;a&gt;&lt;&#47;p&gt;
+This is awesome &lt;a href=&quot;https:&#47;&#47;google.com&quot;&gt;Test&lt;&#47;a&gt;
```

Comment:
Why is it escaped HTML? This is wrong. Google can still not parse it etc.

Reply:
Resolved